README.md

```diff
@@ -69,5 +70,70 @@ Usage:
 ./add_or_update.py [community.json OR official.json] [github/gitlab url OR app name [github/gitlab url OR app name [github/gitlab url OR app name ...]]]
 ```
 
+### How to make my app a High Quality app ?
+
+A High Quality app will be highlighted in the app list and mark as a level 8 app.  
+To become a High Quality app, a package have to follow the following rules:
+
+* The app should already be in the community list for 2 months.
+* The app should be keep up to date, regarding the upstream source. (If it’s possible with our current YunoHost version)
+* The package itself should be up to date regarding the packaging recommendations and helpers.
+* The package should be level 7, at least.
+* The repository should have testing and master branches, at least. The list should point to HEAD, so the list stays up to date.
+* Any modification should be done to the testing branch, and wait at least for one approval for one member of the Apps group. So that we can ensure that there’s nothing in opposition to those criteria. Nor any changes that would harm servers.
```

Comment:
```suggestion
* Any modification should be done to the testing branch, and wait at least for one approval of one member of the Apps group so that we can ensure that there’s nothing in opposition to those criteria, nor any changes that would harm servers.
```